src/process.cr

```diff
@@ -139,6 +140,14 @@ class Process
   # will be closed automatically at the end of the block.
   #
   # Returns the block's value.
+  #```  
+  # someId = "123ABC456DEF"
+  # command = "echo #{someId}"  
+  # output = IO::Memory.new
+  # status = Process.run(command, shell: true, output: output)
+  # pp output.to_s
```

Comment:
Unfortunately this is not a good example, as this is better solved with `Process.run(command) {|proc| pp proc.output.gets_to_end }` or simply ``` pp `#{command}` ```. Secondarily it demonstrates interpolating arguments without proper quoting over the preferred and safer way of just populating `args`. Finally I don't quite see the need for the `command` local variable.

Reply:
Maybe a good idea to make this better would be using `cat` with input and output streams. The example would write to input and read the same content from output.

Reply:
ok thank you, i'll try again